main.tf

```diff
@@ -8,3 +8,15 @@ resource "azurerm_resource_group" "rg" {
 
   tags = var.common_tags
 }
+
+module "redis-cache" {
```

Comment:
don't you need to store the connection string in a key vault as well?

Reply:
yes, added